First/ViewController.swift

```diff
@@ -8,132 +8,116 @@
-            legalTerms.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor),
-            legalTerms.centerXAnchor.constraint(equalTo: view.layoutMarginsGuide.centerXAnchor),
+            blackBackgroundView.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor, constant: kLeftMargin),
+            blackBackgroundView.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor, constant: kRightMargin),
+            blackBackgroundView.topAnchor.constraint(equalTo: logoImageView.bottomAnchor, constant: kTopMargin),
+            blackBackgroundView.bottomAnchor.constraint(lessThanOrEqualTo: view.layoutMarginsGuide.bottomAnchor, constant: kBlackBottomMargin),
+            
+            userNameTextField.heightAnchor.constraint(equalToConstant: kElementsHeight),
+            userNameTextField.leftAnchor.constraint(equalTo: blackBackgroundView.leftAnchor, constant: kLeftMargin),
+            userNameTextField.rightAnchor.constraint(equalTo: blackBackgroundView.rightAnchor, constant: kRightMargin),
+            userNameTextField.topAnchor.constraint(equalTo: blackBackgroundView.topAnchor, constant: kTopMargin),
+            
+            passwordTextField.heightAnchor.constraint(equalToConstant: kElementsHeight),
+            passwordTextField.leftAnchor.constraint(equalTo: blackBackgroundView.leftAnchor, constant: kLeftMargin),
+            passwordTextField.rightAnchor.constraint(equalTo: blackBackgroundView.rightAnchor, constant: kRightMargin),
+            passwordTextField.topAnchor.constraint(equalTo: userNameTextField.bottomAnchor, constant: kTopMargin),
+            
+            loginButton.heightAnchor.constraint(equalToConstant: kElementsHeight),
+            loginButton.leftAnchor.constraint(equalTo: blackBackgroundView.leftAnchor, constant: kLeftMargin),
+            loginButton.rightAnchor.constraint(equalTo: blackBackgroundView.rightAnchor, constant: kRightMargin),
+            loginButton.topAnchor.constraint(equalTo: passwordTextField.bottomAnchor, constant: kTopMargin),
+            
+            troubleButton.heightAnchor.constraint(equalToConstant: kElementsHeight),
+            troubleButton.leftAnchor.constraint(equalTo: blackBackgroundView.leftAnchor, constant: kLeftMargin),
+            troubleButton.rightAnchor.constraint(equalTo: blackBackgroundView.rightAnchor, constant: kRightMargin),
+            troubleButton.topAnchor.constraint(equalTo: loginButton.bottomAnchor, constant: kTopMargin),
+            troubleButton.bottomAnchor.constraint(equalTo: blackBackgroundView.bottomAnchor, constant: kTroubleBottomMargin),
             
+            legalTermsLabel.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor),
+            legalTermsLabel.leftAnchor.constraint(equalTo: view.leftAnchor),
+            legalTermsLabel.rightAnchor.constraint(equalTo: view.rightAnchor),
+            legalTermsLabel.heightAnchor.constraint(equalToConstant: kLegalTermsHeight),
```

Comment:
tu nie podpiales do gory - w przypadku tak "niskiego" widoku nie bedzie roznicy, ale gdybys w przyszlosci mail dodac wiecej komponentow, legalTerms mogloby najechac na komponenty na gorze. mozesz to rozwiazac przez uzycie greater/lessThanOrEqualToConstant zamiast equalToConstant